### ISSUE TITLE
Crash - AnnotatedImageView.java line 129 com.intercom.input.gallery.AnnotatedImageView.createCanvasWithSameBoundsAsRenderedDrawable

### ISSUE THREAD

**Issue by osama-taha** [intercom/intercom-android#411]
## Version info
  
  - Intercom for Android version:  3.2.10
  - Android OS version: 7.0
  - Android Device:  Galaxy S6 Edge

## Expected behavior

No crash

## Actual behavior

crash

## Logs

Fatal Exception: java.lang.OutOfMemoryError: Failed to allocate a 18249996 byte allocation with 4759656 free bytes and 4MB until OOM
       at dalvik.system.VMRuntime.newNonMovableArray(VMRuntime.java)
       at android.graphics.Bitmap.nativeCreate(Bitmap.java)
       at android.graphics.Bitmap.createBitmap(Bitmap.java:977)
       at android.graphics.Bitmap.createBitmap(Bitmap.java:948)
       at android.graphics.Bitmap.createBitmap(Bitmap.java:915)
       at com.intercom.input.gallery.AnnotatedImageView.createCanvasWithSameBoundsAsRenderedDrawable(AnnotatedImageView.java:129)
       at com.intercom.input.gallery.AnnotatedImageView.access$000(AnnotatedImageView.java:21)
       at com.intercom.input.gallery.AnnotatedImageView$1.run(AnnotatedImageView.java:87)
       at android.os.Handler.handleCallback(Handler.java:751)
       at android.os.Handler.dispatchMessage(Handler.java:95)
       at android.os.Looper.loop(Looper.java:154)
       at android.app.ActivityThread.main(ActivityThread.java:6682)
       at java.lang.reflect.Method.invoke(Method.java)
       at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:1520)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1410)

**Comment by ronocod** [comment]
Thanks for reporting this, we'll look into it.

**Comment by osama-taha** [comment]
we have recorded the issue.. this might help you:

https://www.dropbox.com/s/vik8qjfj86olnzk/59a6aeb60c3b5d4b7bb54277.mp4?dl=0

**Comment by bboyle18** [comment]
Hi @osama-taha 👋 
Thanks for sending on the video.
From what I understand, this crash is consistently reproducible?

If so, would you be able to give us a list of steps to re-create the issue please? The quality of the video was not great so it was hard to follow the exact steps.

Thanks

Brian

**Comment by osama-taha** [comment]
Actually it happened with one user, he was trying to attach images into the conversation and then the application crashed.

**Comment by osama-taha** [comment]
Actually, it just happened again with another user ( 8 times ).

A video (This is the best quality we can share):
https://www.dropbox.com/s/v9bs32urxjrocrw/59b7982aa607cb12e44bbc4c.mp4?dl=0

**Comment by ronocod** [comment]
This should be fixed for everyone in the latest release.
